### ISSUE TITLE
add § to keys available for simple modification

### ISSUE THREAD

**Issue by angpan32** [tekezo/Karabiner-Elements#1322]
I would love to be able to remap a key which is not working on my keyboard. As I have a UK apple  keyboard, on the top left I have a most useless key, which allows to type § and ± (available on the US keyboard as option 6 and option shift =). 
This would be the perfect candidate to remap my broken key to, but alas § does not appear in the choice of keys available for a Simple Modification, and I don't have the skills to create a new Complex modification.

So, could you add § and/or ± to the list (in dropdown menu) of keys available for simple modification?

**Comment by nguyenhimself** [comment]
Here you go. Replace "a" and "b" with your preferred keys:

```
{
    "description": "a -> §",
    "manipulators": [
        {
            "from": {
                "key_code": "a",
                "modifiers": {
                    "optional": [
                        "caps_lock"
                    ]
                }
            },
            "to": [
                {
                    "key_code": "6",
                    "modifiers": [
                        "left_option"
                    ]
                }
            ],
            "type": "basic"
        }
    ]
}
```

and:

```
{
    "description": "b -> ±",
    "manipulators": [
        {
            "from": {
                "key_code": "b",
                "modifiers": {
                    "optional": [
                        "caps_lock"
                    ]
                }
            },
            "to": [
                {
                    "key_code": "=",
                    "modifiers": [
                        "left_option",
                        "left_shift"
                    ]
                }
            ],
            "type": "basic"
        }
    ]
}
```

**Comment by nguyenhimself** [comment]
Read this: https://pqrs.org/osx/karabiner/json.html

Yes, it's a bit complicated. It's just that kind of software. Nobody currently has free time to help fix that, including me.

**Comment by angpan32** [comment]
ah, ok, thanks. 

After a struggle, I sorta got it to work...on reverse. The code above allows me to press the key "a" and to type as output "§". I want instead to press § (not "a") and to type tab, in particular so that pressing Cmd+§ would activate the app switcher.
I tried altering the code in the obvious way but it did not work. 
 
For the other rule, I'd like to press simultaneously the two keys control+'  (or option+', either way) to type `

Could you help?

**Comment by nguyenhimself** [comment]
>so that pressing Cmd+§ would activate the app switcher.

Try this (Note: I haven't tested this yet):

```
{
    "description": "§ → Tab",
    "manipulators": [
        {
            "from": {
                "key_code": "6",
                "modifiers": {
                    "mandatory": [
                        "option"
                    ]
                }
            },
            "to": [
                {
                    "key_code": "tab"
                    ]
                }
            ],
            "type": "basic"
        }
    ]
}
```

>For the other rule, I'd like to press simultaneously the two keys control+' (or option+', either way) to type `

Try this (also, not tested):

{
    "description": "Control + ' -> `",
    "manipulators": [
        {
            "from": {
                "key_code": "quote",
                "modifiers": {
                    "mandatory": [
                        "control"
                    ]
                }
            },
            "to": [
                {
                    "key_code": "grave_accent_and_tilde"
                }
            ],
            "type": "basic"
        }
    ]
}

**Comment by fedde-s** [comment]
I believe that it already is available: it's listed under ‘Controls and symbols’ with the confusing name ‘non_us_backslash’.

**Comment by fedde-s** [comment]
I feel like clarifying this shouldn't be too hard even though I personally don't know the code, but nobody seems to have gotten around to it: https://github.com/tekezo/Karabiner-Elements/issues/1065

**Comment by angpan32** [comment]
Thanks! 

Knowing that it was available under  ‘non_us_backslash’, I was able to use a simple modification and now I have my tab replacement working.

I tried the code above but it did not work; more precisely, if I pressed option 6 to write § then it worked and it typed a Tab. However, if I pressed the top left key then this did not type a tab, it typed instead a §, as if I had not set up the complex modification at all

**Comment by fedde-s** [comment]
I just re-checked the simple modifications dropdown in the current 12.1.0 release, and there's still no option that would give a user who isn't a Karabiner-Elements developer any clue that remapping the § key is supported. The root cause of this issue is still present, and I feel it will just keep leading to more issue reports if this one is closed.

**Comment by fedde-s** [comment]
New comments in the closed issue report #1065 show that people do indeed continue to run into this usability problem.

**Comment by kaosbeat** [comment]
Took me a while to find this, so yes this should be fixed. but it seems to work. the "non_us_backslash" is the §,± key on my keyboard. also called the "section" key.
